modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path

```diff
@@ -113,6 +113,26 @@
 	<td>//tr[contains(.,'${key_workflowTask}') and contains(.,'${key_workflowAssetTitle}')]/td[2]//a</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_INPUT</td>
+	<td>//input[contains(@name,'keywords')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_ICON</td>
+	<td>//button[contains(@class, 'btn btn-monospaced btn-unstyled')][2]</td>
+	<td></td>
+</tr>
```

Comment:
You can make the name of those paths more clear, you can use that same path to **both** areas (_Assigned to me / Assigned to my roles_)
Suggestion:
1. MY_WORKFLOW_TASKS_SEARCH_INPUT_FIELD
2. MY_WORKFLOW_TASKS_SEARCH_ICON

Remember to change the names at the created macro (_searchAssignedToMe_)